global merge (reconfiguring vars-structure)

naumvd95 opened the pull request:
https://mirantis.jira.com/browse/FPL-2688
https://mirantis.jira.com/browse/FPL-2685
jjb test passed on all jobs

Commit:
common: "disable jenkins's node" hook is fixed

https://mirantis.jira.com/browse/FPL-2675

naumvd95 committed:
global merge (reconfiguring vars-structure)

https://mirantis.jira.com/browse/FPL-2688
https://mirantis.jira.com/browse/FPL-2685
jjb test passed on all jobs

rkhozinov commented on servers/partner-ci/9.0/contrail.yaml:
```diff
@@ -2,32 +2,25 @@
+            CONTRAIL_VERSION=5.0
+            VSRX_ORIGINAL_IMAGE_PATH=/storage/contrail/vSRX.400.img
+            VSRX_TARGET_IMAGE_PATH=/var/lib/libvirt/images/vSRX.400.img
+            VSRX_TEMPLATE_PATH=/storage/contrail/vSRX_400.template'
```
`VSRX_TEMPLATE_PATH=/storage/contrail/vSRX_400.template'` <<error

rkhozinov replied:
fixed

rkhozinov commented on servers/partner-ci/9.0/contrail.yaml:
```diff
@@ -448,20 +441,23 @@
+              echo $param >> common.properties
+            done
+          fi
+          if [ ! -z "${{SPECIFIC_PROPS}}" ]; then
```
Seems like logic mistake:
If `COMMON_PROPS` doesn't exist, then common.properties won't be erased, and then will be injected with invalid properties. Please clean `*.properties` in for each case

rkhozinov replied:
fixed

rkhozinov commented on servers/partner-ci/9.0/contrail.yaml:
```diff
@@ -448,20 +441,23 @@
     builders:
       - get-refs
+      - shell: |
+          if [ ! -z "${{COMMON_PROPS}}" ]; then
```
the same as below

Commit:
refactoring: updates, inject are moved to common

- updates builder   -> get-updates
- inject properties -> get-props
- slack publisher   -> test-properties
- some typos is fixed

naumvd95 commented on servers/partner-ci/9.0/contrail.yaml:
```diff
@@ -513,42 +497,13 @@
-                  ENABLE_MOS_UBUNTU_SECURITY=true
-      - conditional-step:
-          condition-kind: shell
-          condition-command: |
```
why remove?

rkhozinov commented:
@naumvd95  I've merged #38 to that branch, so please check it also

naumvd95 committed:
jjb tests passed

job-test passed after this changes
http://jenkins-tpi.bud.mirantis.net:8080/job/9.1.manila.bvt.test/30/injectedEnvVars/
